Gems/Atom/Feature/Common/Code/Source/RayTracing/RayTracingAccelerationStructurePass.cpp

```diff
@@ -202,6 +275,85 @@ namespace AZ
             context.GetCommandList()->BuildTopLevelAccelerationStructure(*rayTracingFeatureProcessor->GetTlas());
 
             ++m_frameCount;
+
+            EndScopeQuery(context);
+        }
+
+        void RayTracingAccelerationStructurePass::AddScopeQueryToFrameGraph(AZ::RHI::FrameGraphInterface frameGraph)
+        {
+            const auto addToFrameGraph = [&frameGraph](AZ::RHI::Ptr<AZ::RPI::Query> query)
+            {
+              query->AddToFrameGraph(frameGraph);
+            };
+
+            ExecuteOnTimestampQuery(addToFrameGraph);
+            ExecuteOnPipelineStatisticsQuery(addToFrameGraph);
+        }
+
+        void RayTracingAccelerationStructurePass::BeginScopeQuery(const AZ::RHI::FrameGraphExecuteContext& context)
+        {
+            const auto beginQuery = [&context, this](AZ::RHI::Ptr<AZ::RPI::Query> query)
+            {
+              if (query->BeginQuery(context) == AZ::RPI::QueryResultCode::Fail)
+              {
+                  AZ_UNUSED(this); // Prevent unused warning in release builds
+                  AZ_WarningOnce(
+                      "RayTracingAccelerationStructurePass", false,
+                      "BeginScopeQuery failed. Make sure AddScopeQueryToFrameGraph was called in SetupFrameGraphDependencies"
+                      " for this pass: %s",
+                      this->RTTI_GetTypeName());
+              }
+            };
+
+            if (context.GetCommandListIndex() == 0)
```

Comment:
Do you even need this check? This check was added to RenderPass as you can have multiple command lists encoding gpu commands for one scope. I dont think the this possible for RT acceleration passes.

Reply:
It seems like we do not need this. I have removed this line, along with the other checks pointed out